helm-chart/templates/tests/pod.yaml

```diff
@@ -0,0 +1,44 @@
+{{- if .Values.test.enabled -}}
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    helm.sh/hook: test-success
+    helm.sh/hook-delete-policy: before-hook-creation,hook-succeeded
+  labels: {{- include "json-rpc-relay.labels" . | nindent 4 }}
+  name: {{ include "json-rpc-relay.fullname" . }}-test
+  namespace: {{ include "json-rpc-relay.namespace" . }}
+spec:
+  containers:
+    - name: test
+      image: "{{ .Values.test.image.repository }}:{{ .Values.test.image.tag }}"
+      imagePullPolicy: {{ .Values.test.image.pullPolicy }}
+      args:
+        - run
+        - /test/postman.json
+        - --env-var
+        - baseUrl=http://{{ include "json-rpc-relay.fullname" . }}:{{ .Values.service.port }}
+      securityContext:
+        allowPrivilegeEscalation: false
+        capabilities:
+          drop: [ALL]
+        readOnlyRootFilesystem: true
+      volumeMounts:
+        - name: relay-test
+          mountPath: /test
+          readOnly: true
+  restartPolicy: Never
+  securityContext:
+    fsGroup: 1000
+    runAsGroup: 1000
+    runAsNonRoot: true
+    runAsUser: 1000
+    seccompProfile:
+      type: RuntimeDefault
+  terminationGracePeriodSeconds: 1
```

Comment:
```
  terminationGracePeriodSeconds: {{ .Values.terminationGracePeriodSeconds }}
```
Suggesting using the value in the values.yaml here so the terminationGracePeriod is configurable per installaition.

Reply:
Updated